### PR TITLE
Restore ne 1st Order Admin Lines rule

### PR DESCRIPTION
**Pull request by rmarianski**
Connects to https://github.com/tilezen/vector-datasource/issues/1064

@nvkelso is this what you had in mind in https://github.com/tilezen/vector-datasource/issues/1064#issuecomment-248665808 ?

### CODE REVIEW

File: yaml/boundaries.yaml [hunk 1]
```diff
@@ -52,6 +52,10 @@ filters:
     table: osm
 
   # ne
+  - filter: {featurecla: 1st Order Admin Lines}
```
Comment (nvkelso) [inline yaml/boundaries.yaml]: Combine with line 98 or so where general `region` filter selection is.

**Comment by zerebubuth** [comment]
We should recreate the `integration-test/fixtures/ne_50m_admin_1_states_provinces_lines/992-ne_50m_admin_1_states_provinces_lines-nv-ca.shp` file, as it currently has `featurecla=Admin-1 boundary` whereas the newer shapefiles from S3 have `featurecla=1st Order Admin Lines`.

File: yaml/boundaries.yaml (hunk 1 above)
Comment (rmarianski) [inline yaml/boundaries.yaml]: Updated in d724e05.

**Comment by rmarianski** [comment]
Updated fixture in 0c4b529.

**Pull request merged**